docs/README.md

```diff
@@ -0,0 +1,81 @@
+## ODH-Dashboard Design Improvements
+
+### Overview of Current Design
+
+Dashboard Cards are tiled within the dashboard and have several flavors:
+
+- components (ODHApp)
+- documentation, how-to, tutorial (ODHDoc)
+- getting-started (ODHGettingStarted)
+- quickstarts (QuickStart)
+
+These cards are shown below in the odh-dashboard UI:
+
+![odh-dasboard UI](dashboard.png)
+
+
+The cards are fetched from a local [data directory](https://github.com/red-hat-data-services/odh-dashboard/tree/master/data) when odh-dashboard initializes. 
+The data directory tree is shown below:
+
+```
+data
+├── applications
+│   ├── anaconda-ce.yaml
+│   ├── jupyterhub.yaml
+│   ├── ...
+│   └── watson-studio.yaml
+├── docs
+│   ├── anaconda-tutorial.yaml
+│   ├── auto-ai-experiment-model-tutorial.yaml
+│   ├── ...
+│   └── watson-studio-setup-openscale-how-to.yaml
+├── getting-started
+│   ├── anaconda-ce.md
+│   ├── jupyterhub.md
+│   ├── ...
+│   └── watson-studio.md
+└── quickstarts
+    ├── anaconda-ce-create-jupyter-notebook-quickstart.yaml
+    ├── connect-rhosak-notebook-quickstart.yaml
+    ├── ...
+    └── seldon-deploy-outlier-quickstart.yaml
+```
+
+The odh-dashboard will add application cards to its UI only if the card type (ODHApp) attribute "kfdefApplications" references a KfDef component installed by the ODH Operator.
```

Comment:
It used kdefs, csvs, and validation CMs to determine if the application is enabled, only adding these applications to the Enabled view. All applications are added to the Explore view.

Reply:
thanks. I'll update the comments and correct any logic related to not adding ODHApps to the explore view.